apps/products-feed/src/lib/editor-js-plaintext-renderer.test.ts

```diff
@@ -0,0 +1,63 @@
+import { describe, expect, it, vi } from "vitest";
+import { EditorJsPlaintextRenderer } from "./editor-js-plaintext-renderer";
+
+describe("EditorJsPlaintextRenderer", () => {
+  it("Empty response for invalid input", () => {
+    expect(EditorJsPlaintextRenderer({ stringData: "not json" })).toBe(undefined);
+    expect(EditorJsPlaintextRenderer({ stringData: "" })).toBe(undefined);
+  });
+  it("Paragraph with no formatting", () => {
```

Comment:
```suggestion
  it("returns paragraph with no formatting", () => {
```

I think we can improve the test names, as currently, the cases don't read like sentences. What I propose is introducing some verbs and context. For example, "returns paragraph with no formatting" lacks information about what you need to pass for the function to return that. The proposed version could be "returns paragraph with no formatting when passed a xyz block".

Reply:
Agree, lets work on descriptive tests

Reply:
Good idea. Turns out, if the description is too complicated it's better to write a separate test. Win-win.